partner_aging/wizard/res_partner_aging_supplier.py

```diff
@@ -0,0 +1,367 @@
+# Copyright 2022 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, tools
+
+
+class ResPartnerAgingSupplier(models.Model):
+    _name = "res.partner.aging.supplier"
+    _description = "Res Partner Aging Supplier"
+    _auto = False
+    _order = "partner_id"
+
+    partner_id = fields.Many2one("res.partner", "Partner", readonly=True)
+    max_days_overdue = fields.Integer("Days Outstanding", readonly=True)
+    avg_days_overdue = fields.Integer(readonly=True)
+    date = fields.Date(readonly=True)
+    date_due = fields.Date("Due Date", readonly=True)
+    inv_date_due = fields.Date("Invoice Date", readonly=True)
+    total = fields.Float(readonly=True)
+    not_due = fields.Float("Not Due Yet", readonly=True)
+    days_due_01to30 = fields.Float("1/30", readonly=True)
+    days_due_31to60 = fields.Float("31/60", readonly=True)
+    days_due_61to90 = fields.Float("61/90", readonly=True)
+    days_due_91to120 = fields.Float("91/120", readonly=True)
+    days_due_121togr = fields.Float("+121", readonly=True)
+    invoice_ref = fields.Char("Their Invoice", size=25, readonly=True)
+    invoice_id = fields.Many2one("account.move", "Invoice", readonly=True)
+    salesman = fields.Many2one("res.users", "Sales Rep", readonly=True)
```

Comment:
Here the same

Reply:
The data in this fields doesn't have to be imported in any case. So from my side the actual code is correct.